iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Branch.java

```diff
@@ -179,4 +179,162 @@
     Overload checkTemporaryLimits2(float limitReduction);
 
     Overload checkTemporaryLimits2();
+
+    /**
+     * Get the active power in MW injected at side one of this equipment.
+     * <p>
+     * Depends on the working variant.
+     * @see VariantManager
+     */
+    default double getP1() {
+        return getTerminal1().getP();
+    }
+
+    /**
+     * Set the active power in MW injected at side one of this equipment.
+     * <p>
+     * Depends on the working variant.
+     * @see VariantManager
+     */
+    default Branch<I> setP1(double p) {
```

Comment:
For me, it should return directly `I` not `Branch<I>` (same remark for other setters)